workflows/dev_deploy.yml

```
@@ -0,0 +1,11 @@
+name: UMC Dev CI/CD
+
+on:
+  pull_request:
+    types: [closed]
+  workflow_dispatch: # (2).수동 실행도 가능하도록
+
+jobs:
+  build:
+    runs-on: ubuntu-latest # (3).OS환경
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)